src/apps/concepts/components/ViewConceptsHeader.tsx

```diff
@@ -126,20 +125,13 @@ const ViewConceptsHeader: React.FC<Props> = ({
           open={Boolean(switchSourceAnchor)}
           onClose={handleSwitchSourceClose}
         >
-          <TextField 
-          multiline
-          className={classes.textField} 
-          InputProps={{
-            className: classes.underline
-          }} 
-            inputProps={{
-              className: classes.input
-            }}
-            value={
-              showSources ? "Choose a source/dictionary" : "Select a different source/dictionary"
-            }
-            onClick={() => setShowSources(!showSources)}
-          />
+        <Button 
+           variant="contained" 
```

Comment:
It would be better to use the `text` variant here so it keeps the same UX

```suggestion
           variant="text" 
```